File: Task6/ex3.py
#include<iostream>
using namespace std;
int main() 
{ 
    int a1,b1,a2,b2;
    cin >> a1 >> b1 >> a2 >> b2;
    if(a1*b2>a2*b1) cout<<">";
    else if(a1*b2<a2*b1) cout<<"<";
         else cout<<"=";
}

File: 7class/ex3.cpp
using namespace std;

int main() {

  long long n, i, s;
  cin >> n;
  s=0;
  for (i=1; i<=n; i++)
   {
      s=s+(2*i-1);
      cout << 2*i-1;
      if (i<n) cout << " + ";
   }
  cout << endl << s; 
}